treeherder/seta/job_priorities.py

```diff
@@ -87,7 +55,7 @@ def _gecko_decision_task_request(project):
             if task_request.counter % jp.priority != 0:
                 job_priorities.append(jp)
 
-    return _process(build_system='taskcluster', job_priorities=job_priorities)
+    return _process(project, build_system='taskcluster', job_priorities=job_priorities)
```

Comment:
In the taskcluster decision task code we check to ensure the current repo is supported by seta, and if not then the SETA request is not done. So we will only receive valid project names here. Is that what you mean?